src/dashboard-refactor/styled-components.tsx

```diff
@@ -42,3 +45,35 @@ export const Icon = styled.div<IconProps>`
             animation: ${rotate(-90)} 1s ease-in 1;
         `}
 `
+
+interface LoadingContainerProps {
```

Comment:
It's not clear what the `left`, `bottom`, etc. props mean at a glance. Looking further into it, it should be the responsibility of the component embedding the loading container to put a `<Margin>` element around it.

Reply:
Should I change margin component to `display: flex; align-items: center; justify-content: center`? The reason I've constructed the loading container this way is to avoid having to re-style a margin element just for the loading indicator each time it's used, and I didn't think it made sense to define those styles into the margin element.

Reply:
What I did in `Memex-Social` is have a `LoadingIndictor` which is just the spinner, and a `LoadingScreen` which is 100% width, centers children, etc. containing a `Margin` containing the `LoadingIndicator`.

Reply:
Cool cool I'll follow that pattern, cheers.